libs/platform/src/lib/form/form-group/form-group-header/form-group-header.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { FieldGroup } from '../../form-helpers';
+import { HintOptions } from '@fundamental-ngx/platform/shared';
+
+@Component({
+    // eslint-disable-next-line @angular-eslint/component-selector
+    selector: '[fdp-form-group-header]',
```

Comment:
I think platform is more component-style library, so using "directive-selector" approach here might be wrong.

Reply:
That component is for internal use, so user will not have any interaction with it. For us, it will reduce dummy tags on DOM